src/mixins/amalgamation-mixin.ts

```diff
@@ -248,15 +248,6 @@ export default class AmalgamationMixin extends Vue {
     return this.getAmalgamatingBusinesses.every(business => (business.type === AmlTypes.FOREIGN))
   }
 
-  /** True if all companies in the table are foreign or extra-provincial. */
-  get isAllForeignOrEp (): boolean {
```

Comment:
I think this rule is obsolete as Extra Pro A companies are not LEAR anymore (FOREIGN). Please correct me if I'm wrong.

Reply:
I agree.